q2_alignment/_mafft.py

```diff
@@ -24,10 +24,10 @@ def run_command(cmd, output_fp, verbose=True):
         subprocess.run(cmd, stdout=output_f, check=True)
 
 
-def mafft(sequences: DNAFASTAFormat) -> AlignedDNAFASTAFormat:
+def mafft(sequences: DNAFASTAFormat, threads: int=1) -> AlignedDNAFASTAFormat:
```

Comment:
Could you call the parameter `n-threads` for consistency with some of the other q2 methods that support multi-threaded mode (e.g., `dada2 denoise-*`)?